remappings.txt

```diff
@@ -1,6 +1,6 @@
-@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/
-@prb/math/=lib/prb-math/
-@prb/test/=lib/prb-test/src/
-forge-std/=lib/forge-std/src/
-solady/=lib/solady/src/
-solarray/=lib/solarray/src/
+@openzeppelin/contracts/=node_modules/@openzeppelin/contracts/
+@prb/math/=node_modules/@prb/math/
+@prb/test/=node_modules/@prb/test/
+forge-std/=node_modules/forge-std/
+solady/=node_modules/solady/
+solarray/=node_modules/solarray/
```

Comment:
Will these work with pnpm node modules system, @andreivladbrg? See what I asked you [here}(https://github.com/sablier-labs/v2-core/discussions/508#discussioncomment-7751667)

Can you please perform some tests?

Reply:
I think you got my point wrong, as also said in that discussion, pnpm places only the third party deps under the `node_modules/.pnpm`.

If you clone the repo, you can run `pnpm install` and its deps would be place under `node_modules/`, so we don't have to worry about the remappings for the projects itself. 

What I mean when I say that we should recommend `yarn add`  instead of `pnpm add` is when an integrator (as allo was) should add to their project our package.  For them, when adding our package, the third party deps are `openzeppelin/contracts/` and `@prb/math/`:

https://github.com/sablier-labs/v2-core/blob/96f89cfabea3dc6638f0b64a30cfb44066a95944/package.json#L13-L16

More explantions about CLIs: 
- `pnpm add` : Installs a package and any packages that it depends on. https://pnpm.io/cli/install
- `pnpm install` : is used to install all dependencies for a project. https://pnpm.io/cli/add


> Can you please perform some tests? 

Did several tests when I've written my answer [here](https://github.com/sablier-labs/v2-core/discussions/508#discussioncomment-7374482).

Reply:
> I think you got my point wrong

I didn't; I was just unclear in my question.

> If you clone the repo, you can run pnpm install and its deps would be place under node_modules/, so we don't have to worry about the remappings for the projects itself.

I know. I was referring to third-party developers installing V2 Core. Would the remappings still work recursively?

Sorry for not being clear earlier.

> What I mean when I say that we should recommend `yarn add` instead of `pnpm add` is when an integrator

This is equivalent to asking the integrators to switch over to Pnpm instead of Yarn. And, as I've asked you in the discussion: `Surely there must be a way to force Pnpm to not put the dependencies under `node_modules/.pnpm` without asking users to manually install those packages?`

> Did several tests when I've written my answer https://github.com/sablier-labs/v2-core/discussions/508#discussioncomment-7374482.

Thanks but you missed one of my questions. Let's follow up there.